nginx-nodejs-redis/README.md

```diff
@@ -0,0 +1,119 @@
+## Compose sample application
+
+## Node.js application with Nginx proxy and Redis database
+
+Project structure:
+```
+.
+├── docker-compose.yml
+├── nginx
+│   ├── Dockerfile
+│   └── nginx.conf
+├── web
+│   ├── Dockerfile
+│   ├── package.json
+│   └── server.js
+├── web1
+│   ├── Dockerfile
+│   ├── package.json
+│   └── server.js
+└── web2
+    ├── Dockerfile
+    ├── package.json
+    └── server.js
+
+4 directories, 12 files
+```
+
+[_docker-compose.yml_](docker-compose.yml)
+```
+redis:
+    image: 'redislabs/redismod'
+    ports:
+      - '6379:6379'
+  web1:
+    restart: on-failure
+    build: ./web1
+    ports:
+      - '81:5000'
+  web2:
+    restart: on-failure
+    build: ./web2
+    ports:
+      - '82:5000'
+  nginx:
+    build: ./nginx
+    ports:
+    - '80:80'
+    depends_on:
+    - web1
+    - web2
```

Comment:
Should be update regarding the modification asked in the `docker-compose.yml` file